click_odoo_contrib/update.py

```diff
@@ -88,13 +198,46 @@ def OdooEnvironmentWithUpdate(database, ctx, **kwargs):
                 "Updating addons for their hash changed: %s.",
                 ",".join(to_update.keys()),
             )
-    if ctx.params["i18n_overwrite"]:
-        odoo.tools.config["overwrite_existing_translations"] = True
+    odoo.tools.config["overwrite_existing_translations"] = ctx.params["i18n_overwrite"]
```

Comment:
@Yajo not sure about this one. Can you make another PR for this line?

Reply:
Yes, this was more a "BTW" than anything else.